src/test/java/uk/gov/hmcts/reform/idam/util/SecurityUtilTest.java

```diff
@@ -18,6 +19,7 @@
 
 
 @ExtendWith(MockitoExtension.class)
+@SuppressWarnings({"PMD.UnnecessarySemicolon"})
```

Comment:
Why can't you delete that unnecessary semicolon?

Reply:
Semicolon is removed